tools/fqltool/src/org/apache/cassandra/fqltool/commands/Replay.java

```diff
@@ -79,15 +79,15 @@ public void run()
                 if (!basePath.exists() || !basePath.isDirectory())
                 {
                     System.err.println("The results path (" + basePath + ") should be an existing directory");
-                    System.exit(1);
+                    throw new IllegalArgumentException("The results path (" + basePath + ") should be an existing directory");
                 }
                 resultPaths = targetHosts.stream().map(target -> new File(basePath, target)).collect(Collectors.toList());
                 resultPaths.forEach(File::mkdir);
             }
             if (targetHosts.size() < 1)
             {
                 System.err.println("You need to state at least one --target host to replay the query against");
-                System.exit(1);
+                throw new IllegalArgumentException("You need to state at least one --target host to replay the query against");
```

Comment:
is it possible to not duplicate the message in error printing and in the exception? (here and other similar places)

Reply:
Fixed.